packages/manager/apps/pci-savings-plan/public/translations/listing/Messages_en_GB.json

```diff
@@ -21,10 +21,11 @@
   "order_instance": "Create an instance",
   "order_rancher": "Get started with Managed Rancher Services",
   "listing_resultats": "results",
-  "months": "{{count}} months",
+  "months": "{{count}} month",
   "months_plural": "{{count}} months",
```

Comment:
why in some cases this key `months_plural` is still here and other cases it is deleted (replaced by `months_other`) ? also would it make more sense to have the singular key which is currently `months`  as `month` instead?

Reply:
You are right, plural should be removed, work will be added by @lionel95200x when he is coming back